### ISSUE TITLE
[v9] Can't import from `firebase/messaging/sw` with Webpack 5 build

### ISSUE THREAD

**Issue by jhuleatt** [firebase/firebase-js-sdk#5314]
### [REQUIRED] Describe your environment

  * Operating System version: Mac OS 11.5.1
  * Browser version: Chrome 
  * Firebase SDK version: 9.0.0-beta.8
  * Firebase Product: messaging

### [REQUIRED] Describe the problem

When I try to import the [service worker version of `getMessaging`](https://firebase.google.com/docs/reference/js/v9/messaging_sw.md#getmessaging):

```js
import { getMessaging } from 'firebase/messaging/sw'; 
```

Webpack gives me the following error:

```
ERROR in ./node_modules/firebase/messaging/sw/dist/index.esm.js 1:0-75
Module not found: Error: Can't resolve '@firebase/messaging/sw' in '/Users/jhuleatt/git/fcm-v9-repro/node_modules/firebase/messaging/sw/dist'
Did you mean 'index.sw.esm2017.js'?
```

#### Steps to reproduce:

1. Clone https://github.com/jhuleatt/fcm-v9-repro
     * [`src/firebase-messaging-sw.js`](https://github.com/jhuleatt/fcm-v9-repro/blob/main/src/firebase-messaging-sw.js) contains the `import` statement
     * [`webpack.config.js`](https://github.com/jhuleatt/fcm-v9-repro/blob/main/webpack.config.js) contains a very basic webpack config
2. Run `npm install`
3. Run `npm run build`
4. You'll see that `src/index.js` compiles without error, but `src/firebase-messaging-sw.js` gives the above error.

**Comment by jhuleatt** [comment]
Workaround is to add:

```js
module: {
  rules: [{
    test: /\.m?js/,
    resolve: {
      fullySpecified: false
    }
  }]
}
```

to Webpack config

**Comment by jonathan-hetrafi** [comment]
> Workaround is to add:
> 
> ```js
> module: {
>   rules: [{
>     test: /\.m?js/,
>     resolve: {
>       fullySpecified: false
>     }
>   }]
> }
> ```
> 
> to Webpack config

still can't find module :'(

**Comment by Feiyang1** [comment]
Fixed in 9.0.0-202172321475
